<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1076</width>
    <height>697</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1076</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionConfig"/>
   </widget>
   <widget class="QMenu" name="menuAnalysis">
    <property name="title">
     <string>Analysis</string>
    </property>
    <addaction name="actionLayout"/>
    <addaction name="actionFirst_Order_Data"/>
    <addaction name="actionParaxial_Ray_Trace"/>
    <addaction name="actionTransverse_Ray_Fan"/>
    <addaction name="actionReal_Ray_Trace"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAnalysis"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionLayout">
   <property name="text">
    <string>Layout</string>
   </property>
  </action>
  <action name="actionTransverse_Ray_Fan">
   <property name="text">
    <string>Transverse Ray Fan</string>
   </property>
  </action>
  <action name="actionLongitudinal_Ray_Fan">
   <property name="text">
    <string>Longitudinal Ray Fan</string>
   </property>
  </action>
  <action name="actionSpot_Diagram">
   <property name="text">
    <string>Spot Diagram</string>
   </property>
  </action>
  <action name="actionFirst_Order_Data">
   <property name="text">
    <string>First Order Data</string>
   </property>
  </action>
  <action name="actionParaxial_Ray_Trace">
   <property name="text">
    <string>Paraxial Ray Trace</string>
   </property>
  </action>
  <action name="actionLog">
   <property name="text">
    <string>Log</string>
   </property>
  </action>
  <action name="actionReal_Ray_Trace">
   <property name="text">
    <string>Single Ray Trace</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About Geopter</string>
   </property>
  </action>
  <action name="actionConfig">
   <property name="text">
    <string>Glass Catalog</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
